pkgs/applications/networking/cluster/terraform/default.nix

```
@@ -34,13 +35,35 @@ let
           --replace "/bin/stty" "${coreutils}/bin/stty"
       '';
 
+      nativeBuildInputs = [ installShellFiles ];
+
+      # https://github.com/posener/complete/blob/9a4745ac49b29530e07dc2581745a218b646b7a3/cmd/install/bash.go#L8
+      completionBash = "complete -C terraform terraform\n";
+      # https://github.com/posener/complete/blob/9a4745ac49b29530e07dc2581745a218b646b7a3/cmd/install/zsh.go
+      completionZsh =  "autoload -U +X bashcompinit && bashcompinit && complete -C terraform terraform\n";
```

Upstream sets nospace for zsh.

```suggestion
      completionZsh =  "autoload -U +X bashcompinit && bashcompinit && complete -o nospace -C terraform terraform\n";
```

---

Changed!